py/server/deephaven/table.py

```diff
@@ -352,8 +369,7 @@ def update(self, formulas: Union[str, Sequence[str]]) -> Table:
         """
         try:
             formulas = to_sequence(formulas)
-            _set_query_scope()
-            with auto_locking_ctx(self):
+            with auto_locking_ctx(self), _query_scope_ctx():
```

Comment:
Is there an implied ordering here? I would prefer to hold the lock for the shortest possible window.